src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -51,17 +53,17 @@ public void storeMessage(Message message) {
   }
 
   /**
-   * Gets messages posted by a specific user.
+   * Gets messages received by a specific user.
```

Comment:
I think it might make sense for this to be the list of all messages that are visible to the given user (both those received by that user as well as messages that the user has written). Otherwise, how does the user view the messages that they have sent or posted publicly?